### DISCUSSION TITLE
[Feature] Share TAGs

### DISCUSSION THREAD

**Discussion by sietzebakker** [immich-app/immich#14601] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Is it also possible (to create) that you can share on TAGs? E.g. on face tags? I have 8 grandchildren. To make all the albums for that is a lot of work. But to share each grandchild his own tag I'm done much faster 😊

### Platform

- [ ] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#6755
